models/token.go

```diff
@@ -21,11 +21,11 @@ type AccessToken struct {
 	Sha1 string `xorm:"UNIQUE VARCHAR(40)"`
 
 	Created           time.Time `xorm:"-"`
-	CreatedUnix       int64
+	CreatedUnix       int64     `xorm:"INDEX"`
```

Comment:
I don't think we ever issue SQL queries that use the `created_unix` column of this table. Will adding an index here help us?